packages/admin/src/form/FieldContainer.tsx

```diff
@@ -118,6 +122,13 @@ export const FieldContainerComponent: React.FC<WithStyles<typeof styles, true> &
         }
     }, [scrollTo]);
 
+    const childrenWithDisabledProp = React.Children.map(children, (child) => {
+        if (React.isValidElement(child)) {
+            return React.cloneElement(child, { disabled });
+        }
+        return child;
+    });
+
```

Comment:
The `disabled` prop wasn't actually used for anything until now. 
To actually disable the input, the input needed it's own `disabled` prop, which could not influence the label-styling.

When using `Field` inside a final-form, there only needs to be one `disabled` prop, as it is passed to the `component`/`children`:

```tsx
<Field label="Text" name="text" component={FinalFormInput} disabled />

// or

<Field name="text" label="Text" disabled>
    {(props) => <FinalFormInput {...props} />}
</Field>
```
 
I now added a similar behaviour, by passing the disabled prop to every child component. 

I'm not sure if this is a good approach:
- This doesn't work when nesting the input within another element 
- and since `FieldContainer` is used within `Field` which might cause some issues

Is there a better way of doing this?
Or are we okay with needing to set multiple `disabled` props when using Input's outside of a final-form?

```tsx
<FieldContainer label="Text" disabled>
    <InputBase onChange={handleChange} value={value} placeholder="Placeholder" disabled />
</FieldContainer>
```

Reply:
(sorry for the late reply)

I would avoid using `React.cloneElement`, espacially when it only solves an issue we have outside of final-form (which is rather rare)